### PR TITLE
SW-5122 Make the ToDo name fluidly shrink with an ellipsis

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
* **#2522** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2522?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/Home/ParticipantHomeView/index.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const ParticipantHomeView = () => {
 
   return (
     <PageWithModuleTimeline>
-      <Grid display={'flex'} flexDirection={'column'} flexGrow={1}>
+      <Grid display={'flex'} flexDirection={'column'} width={'100%'}>
```
Comment (nickgraz) [inline src/scenes/Home/ParticipantHomeView/index.tsx:17]: @karthikbtf unfortunately I had to put this back :( - the container wasn't shrinking correctly at lower resolutions.

Comment (karthikbtf) [reply src/scenes/Home/ParticipantHomeView/index.tsx:17]: No worries, thanks for givint it a try earlier. This is totally fine if it solves the problem.

**Review by karthikbtf — APPROVED** [review]
Looks good to me 👍

**Pull request merged**